runtime/bindings/python/src/openvino/helpers.py

```diff
@@ -0,0 +1,14 @@
+# Copyright (C) 2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+"""openvino helper functions."""
+
+from openvino import Function
+from openvino.inference_engine import IENetwork
+
+
+# TODO: will be removed after full updating MO with new api (cvs-69196)
+def function_to_cnn(ng_function: Function) -> IENetwork:
+    """Get Inference Engine CNN network from openvino function."""
+    capsule = Function.to_capsule(ng_function)
+    return IENetwork(capsule)
```

Comment:
Does it work well? I'm getting error "PyCapsule_GetPointer called with incorrect name" at ie_api.pyx when trying to convert serialize PaddlePaddle model...

Reply:
removed